samples/bluetooth/direction_finding_connectionless_tx/sample.yaml

```diff
@@ -0,0 +1,11 @@
+sample:
+  name: Direction Finding Connectionless Beacon
+  description: Sample application showing connectionless Direction Finding transmission
+tests:
+  sample.bluetooth.direction_finding_connectionless:
+    harness: bluetooth
+    platform_allow: nrf52833dk_nrf52833
+    tags: bluetooth
+  sample.bluetooth.direction_finding_connectionless.aoa:
+    extra_args: OVERLAY_CONFIG="overlay-aoa.conf"
+    platform_allow: nrf52833dk_nrf52833
```

Comment:
Add harness and tags (for consistency with previous test)?

Reply:
added missing tags

Reply:
added missing tags